package/Dockerfile.base

```diff
@@ -0,0 +1,4 @@
+# This Dockerfile is responsible for providing all tooling required to run
```

Comment:
We don't need to include the example base image as a part of manager code. It can be put at e.g. https://github.com/rancher/longhorn-tests/tree/master/manager/test_containers instead.